tests/test_runner/tests.py

```diff
@@ -407,3 +410,50 @@ def test_empty_default_database(self):
         connection = testcases.connections[db.utils.DEFAULT_DB_ALIAS]
         self.assertEqual(connection.settings_dict['ENGINE'], 'django.db.backends.dummy')
         connections_support_transactions()
+
+
+class PersonAdmin(admin.ModelAdmin):
+    def check(self, **kwargs):
+        return [
+            checks.Error(
+                'A check that errors.',
+                obj=Person,
+                id='admin.T001',
+            )
+        ]
```

Comment:
You can probably simulate that with having `run_checks` raise a `SystemCheckError` error instead.

Reply:
Good idea.  The existing test is smaller this way.